tests/tracer/test_propagation.py

```diff
@@ -444,7 +444,7 @@ def test_tracecontext_get_sampling_priority(sampling_priority_tp, sampling_prior
             None,
         ),
         (
-            "01-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01",
+            "01-4bf92f3577b34da6a3ce929d0e0e4736-00f067aa0ba902b7-01-what-the-future-looks-like",
```

Comment:
Can we add a comment that `01` is required for this to work?

Also, we might want to add a test case for when we have version `00` and extra at the end.